lib/util.ts

```diff
@@ -27,29 +35,34 @@ class fixedTFramedTransport extends thrift.TFramedTransport {
   */
 
 const previousPageLocation = parquet_thrift.PageLocation.prototype;
+//@ts-ignore
```

Comment:
I haven't been following along on these conversions - are we just using `@ts-ignore` for now?  Is there a plan to convert these to proper TypeScript?  @wilwade